openshift/system/Dockerfile

```diff
@@ -10,7 +10,6 @@ ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}" \
 
 ENV TZ=:/etc/localtime \
     BUNDLE_GEMFILE=Gemfile \
-    BUNDLE_WITHOUT=development:test \
```

Comment:
Why removing this?

Reply:
obsolete way to control what to exclude, now using `bundle config set`